controllernav: { 
  Gp: 0.2,
  Gi: 0.0,
  Gd: 0.0,
  Gc: 0.0,
  control_freq: 50,
  control_time: 0.02,
  satMin: 40.0,
  satMax: 150.0,
  linearSpeed: 0.25
}
